### DISCUSSION TITLE
RTKQuery with Storybook

### DISCUSSION THREAD

**Discussion by r4hulp** [reduxjs/redux-toolkit#2178] [Q&A]
I am implementing RTKQuery in one of my projects. We are planning to use Storybook to document our UI components. While we are trying to keep our components as independent as possible, there are some scenarios where we need to make RTK Query calls (mostly Fetch queries)...

How can we make this work with Storybook? I see articles around having `slices` with storybook but not `createAPI` mocks. 

Any help will be great.

**Comment by phryneas** [comment]
Just like with tests, I would recommend just using your normal Redux store and mocking an api with https://github.com/mswjs/msw
